test/cli_start_test.go

```diff
@@ -296,3 +296,21 @@ func (suite *PouchStartSuite) TestStartWithPidsLimit(c *check.C) {
 
 	command.PouchRun("start", name).Assert(c, icmd.Success)
 }
+
+// TestStartMultiContainers tries to start more than one container.
+func (suite *PouchStartSuite) TestStartMultiContainers(c *check.C) {
+	name1 := "TestStartMultiContainer-1"
+	name2 := "TestStartMultiContainer-2"
+
+	res1 := command.PouchRun("create", "--name", name1, busyboxImage, "top")
+	defer DelContainerForceMultyTime(c, name1)
+	res1.Assert(c, icmd.Success)
+
+	res2 := command.PouchRun("create", "--name", name2, busyboxImage, "top")
+	defer DelContainerForceMultyTime(c, name2)
+	res2.Assert(c, icmd.Success)
+
```

Comment:
A ```for``` circulation can be used here to enhance the degree of simplicity.  Just like the way you do in ```TestUnpauseMultiContainers```